main.go

```diff
@@ -330,34 +329,27 @@ func main() {
 		cancel()
 	}()
 
-	sqlDB, err := models.OpenDB(*dbNameF, *dbUsernameF, *dbPasswordF, l.Debugf)
-	if err != nil {
-		l.Panicf("Failed to connect to database: %+v", err)
-	}
-	defer sqlDB.Close()
-	db := reform.NewDB(sqlDB, reformMySQL.Dialect, nil)
-
 	postgresDB, err := models.OpenPostgresDB(*postgresDBNameF, *postgresDBUsernameF, *postgresDBPasswordF, l.Debugf)
```

Comment:
We could also rename that function to `OpenDB` to avoid renames everywhere else.